Rakefile

```diff
@@ -82,6 +82,26 @@ BASH
 
 end
 
+# no desc, doesn't really work outside of rake tasks, can't say: `rake gimme; go` as the vars aren't exported
+task :gimme do
+    go_version = "1.18"
+
+    if (`which gimme`; $?.success?)
+      sh "GIMME_DOWNLOAD_BASE=https://binaries.ddbuild.io/golang/ gimme #{go_version}"
```

Comment:
https://binaries.ddbuild.io/golang/ is not something that would work for external contributors, so I don't think we should override `GIMME_DOWNLOAD_BASE` like this.